Engine/Colliders/CircleCollider.cs

```diff
@@ -8,12 +10,16 @@ namespace AntEngine.Colliders
     public class CircleCollider : Collider
     {
         public CircleCollider(Transform transform, Transform parentTransform) : base(transform, parentTransform) {}
-        
-        //TODO : Add attributes
+
+        /// <summary>
+        /// Radius of the circle. This is the maximum component of the scale vector. (we're not handling ellipses)
+        /// </summary>
+        public float Radius => MathF.Max(ColliderTransform.Scale.X, ColliderTransform.Scale.Y);
+
         public override bool checkCollision(CircleCollider circleCollider)
         {
-            //TODO : Implement
-            throw new System.NotImplementedException();
+            float distance = Vector2.Distance(ColliderTransform.Position, circleCollider.ColliderTransform.Position);
+            return distance > Radius + circleCollider.Radius;
```

Comment:
Did you fix the part you told me you had to fix?

Reply:
Noooope 👀